### PR TITLE
winpr: add intrin.h

### PR DESCRIPTION
**Pull request by bmiklautz**
Exposing lzcnt in crt.h might cause compiler errors (redefinition) with
recent versions of gcc (>=4.9) when winpr is included in other projects.
As lzcnt isn't part of crt according to MSDN and also shouldn't be
exported by default it was moved to intrin.h.

The related test was also moved to the top level directory of winpr.

### CODE REVIEW

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by mfleisz** [comment]
+1

File: winpr/test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+set(MODULE_NAME "TestWinpr")
```
Comment (nfedera) [inline winpr/test/CMakeLists.txt]: I don't really care but the other test is called TestWinPRUtils (uppercase PR).

**Comment by nfedera** [comment]
Not your code but winpr/test/TestIntrinsics.c is broken because TestIntrinsics() always returns 0.

**Pull request merged**